### ISSUE TITLE
Add OpenDyslexic Support

### ISSUE THREAD

**Issue by bkerensa** [Glucosio/glucosio-android#190]
Add OpenDyslexic Font support which can be toggled on via Accessibility submenu for people with dyslexia.

See http://opendyslexic.org/

**Comment by bkerensa** [comment]
This should be resolved

**Comment by PaoloRotolo** [comment]
Yes, closing...
